.github/workflows/github-actions-test.yml

```diff
@@ -0,0 +1,28 @@
+name: test
+on : [pull_request, push]
+jobs:
+  Explore-Github-Actions:
+    runs-on: macos-latest
+    steps:
+      - run: echo "The name of your branch is ${{ github.ref }} and your repository is ${{ github.repository }}."
+      - name: Checkout repository code
+        uses: actions/checkout@v2
+
+      - name: Setup python
+        uses: actions/setup-python@v2
+        with: 
+          python-version: '3.7.7'
+          architecture: 'x64'
+      
+      - name: Install python packages, build and test
+        run: |
+          python -m pip install --upgrade pip
+          pip install virtualenv
+          python -m venv virtualenv-ml-conversational
+          source ./virtualenv-ml-conversational/bin/activate
+          pip install --upgrade pip
+          pip install -r requirements.txt
```

Comment:
It might be better to extract the dependency installation as a separate step. 

- steps
  - checkout
  - setup
  - install dependencies
  - test

Reply:
Done!! Updated in the latest change